### ISSUE TITLE
save_file not working correctly with dompdf.php ?

### ISSUE THREAD

**Issue by PhenX** [dompdf/dompdf#178]
_Original author: stu.tipp...@gmail.com (August 10, 2010 12:19:33)_

I am trying to use the 'output_file' and 'save_file' variables as directed on http://code.google.com/p/dompdf/wiki/Usage#Invoking_dompdf_via_the_web 

What I'm aiming to do is to save a file as a pdf on my server, rather than downloading it. So instead of using the url: dompdf.php?base_path=www%2Ftest%2F&amp;input_file=image.html

I am trying to use: dompdf.php?base_path=www%2Ftest%2F&amp;input_file=image.html&amp;output_file=image.pdf&amp;save_file=true

However, instead of saving the file, it gives me a download box again as if I was going to the original url, and it's not taking into account the 'save_file' variable at all.

I'm using the latest dompdf_0-6-0_beta1 release.

Basically, I'm trying to save a pdf with images onto my server. I can save a pdf fine by using the code here: http://code.google.com/p/dompdf/wiki/Usage#Using_the_dompdf_class_directly

However, the code in the url above blocks out any images (even local ones), which going directly to: dompdf.php?base_path=www%2Ftest%2F&amp;input_file=image.html does not.

_Original issue: http://code.google.com/p/dompdf/issues/detail?id=172_
